runtimebp/config.go

```diff
@@ -37,5 +37,5 @@ func InitFromConfig(cfg Config) {
 		min = cfg.NumProcesses.Min
 	}
 	prev, current := GOMAXPROCS(min, max)
-	fmt.Fprintf(os.Stderr, "GOMAXPROCS: %d %d\n", prev, current)
+	fmt.Fprintf(os.Stderr, "GOMAXPROCS: previous: %d current: %d\n", prev, current)
```

Comment:
🔕 or simply old/new (we actually wanted to use `old`/`new` as the variable names instead of `prev`/`current`, but `new` is a keyword in go so we have to pick our second choices 😅)